src/components/BaseHeading/BaseHeading.tsx

```diff
@@ -0,0 +1,26 @@
+import './BaseHeading.css';
+
+interface BaseHeadingProps {
+  children?: React.ReactNode;
+  size?: 'xl' | 'l' | 'm' | 's';
```

Comment:
Actualizar los posibles valores por una version de 2 caracteres, y a su vez tambien realizar el cambio en el archivo css para que haga match con estos nuevos nombres

```suggestion
  size?: 'xl' | 'lg' | 'md' | 'sm';
```